src/Controls/src/Core/Window/Window.cs

```diff
@@ -199,7 +200,7 @@ public double MinimumHeight
 		}
 
 		int _batchFrameUpdate = 0;
-
+		bool disableHandlerUpdate;
```

Comment:
I chatted with @mattleibow about this one a bit, and I think my suggestions are somewhat just digging us into a deeper hole here :-)

I think we've been trying to fix this issue the wrong way for some time now and need to revert this entire method back to what it originally looked like here

https://github.com/dotnet/maui/pull/4942/files#diff-07d5a2c0b5069d448623e88d890e41bc2c21e12f99bd01f1349903a332d4130cR202-R222

And then change this method 
```C#
private protected override void UpdateHandlerValue(string property)
		{
			if (disableHandlerUpdate)
				return;

			base.UpdateHandlerValue(property);
		}
```

to

```C#
private protected override void UpdateHandlerValue(string property)
		{
			if (_batchFrameUpdate > 0)
				return;

			base.UpdateHandlerValue(property);
		}
```